Multisig/UI/Settings/SafeSettingsViewController/SafeSettingsViewController.swift

```diff
@@ -21,9 +21,12 @@ class SafeSettingsViewController: LoadableViewController, UITableViewDelegate, U
     private var currentDataTask: URLSessionTask?
     private var sections = [SectionItems]()
     private var safe: Safe?
+    private var safeOwners: [AddressInfo] = []
```

Comment:
comment why we are loading it here instead of relying on safe info - would help for the future code readers